test/integration/targets/vmware_guest_cross_vc_clone/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+# Test code for the vmware_guest_serial_port Operations.
+# Copyright: (c) 2020, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_datacenter: true
+
+- name: clone a template across VC
+  vmware_guest_cross_vc_clone:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: "{{ vcenter_password }}"
+    validate_certs: no
+    name: "test_vm"
+    destination_vm_name: "cloned_vm_from_template"
+    destination_vcenter: '{{ destination_vcenter_hostname }}'
```

Comment:
I don't see the variables defined. Am I missing here something?

Reply:
I am actually not sure how to use `'{{ destination_vcenter_hostname }}'` 
That's why I have marked the integration test as `unsupported`.
If you or @goneri can point me to the right variables, that will be helpful

Reply:
You need to define the variable in `vars` directory of `vmware_guest_cross_vc_clone` role. 

@goneri Do we have a plan to have multiple vCenter in zuul environment?

Reply:
Today the vcenter VM hostname is `vcenter`, and we cannot easily rename an instance. So if we want a second vcenter, we will probably need a second VM image. It's a bit cumbersome.

Reply:
Thanks for your reply. I will keep it `unsupported` for now then.